module/callback_handlers.py

```diff
@@ -9,6 +9,8 @@
 # Needed to correctly run functions using globals()
 from module.utils.multi_lang_utils import get_on_demand_text
 
+from module.commands.esami import *  # pylint: disable=wildcard-import,unused-wildcard-import
+from module.commands.lezioni import *  # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.help import * # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.drive_contribute import * # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.aulario import * # pylint: disable=wildcard-import,unused-wildcard-import
```

Comment:
you should not use `from something import *`

Reply:
Okay, I'll try to specify which method is actually needed